spec/support/ansible_shared/provider.rb

```diff
@@ -3,15 +3,18 @@
 shared_examples_for "ansible provider" do
   describe "#connect" do
     let(:attrs) { {:username => "admin", :password => "smartvm", :verify_ssl => OpenSSL::SSL::VERIFY_PEER} }
+    let(:miq_server) { FactoryGirl.create(:miq_server) }
```

Comment:
These changes are embedded specific, so I feel like they belong in the embedded provider spec rather than in the shared examples.